src/test/java/org/jabref/gui/search/GlobalSearchBarTest.java

```diff
@@ -88,8 +89,8 @@ void recordingSearchQueriesOnFocusLostOnly(FxRobot robot) throws InterruptedExce
 
         // Set the focus to another node to trigger the listener and finally record the query.
         DefaultTaskExecutor.runInJavaFXThread(() -> hBox.requestFocus());
-        List<String> lastSearchHistory = stateManager.getWholeSearchHistory();
-        List<String> expected = List.of("Smith");
+        ObservableList<String> lastSearchHistory = stateManager.getWholeSearchHistory();
+        ObservableList<String> expected = FXCollections.observableArrayList("Smith");
```

Comment:
The type is still `ObservableListWrapper`.

org.opentest4j.AssertionFailedError: expected: com.sun.javafx.collections.ObservableListWrapper@105ad36f<[Smith]> but was: com.sun.javafx.collections.ObservableListWrapper@dfc809b<[Smith]>.

I still think, ´stateManager.getWholeSearchHistory()` should be converted to a `List`. (For instance, using `.stream().toList()`.

Reply:
Using `stream().toList()` causes issues and doesn't return the search history elements as expected.